### PR TITLE
Home vNext 1: restructure navigation

### PR DESCRIPTION
**Pull request by mdjastrzebski**
Part of Home vNext feature

Scope:
- [x] Remove tab navigation & restructure top level navigation
- [x] Provide temporary home content and navigation buttons until proper parts are implemented

### CODE REVIEW

File: src/components/modern/Layout.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// @flow
```
Comment (jdnmr) [inline src/components/modern/Layout.js:1]: No license info etc.?

File: src/screens/Home/Home.js [hunk 2]
```diff
@@ -1,788 +1,86 @@
...
-  }
+      <Content>
+        {/* Temporary content */}
+        <MainContent>
+          {/* eslint-disable-next-line i18next/no-literal-string */}
+          <Text>Hello Pillar!</Text>
+        </MainContent>
+
+        {/* Temporary navigation section */}
+        {/* eslint-disable-next-line i18next/no-literal-string */}
```
Comment (jdnmr) [inline src/screens/Home/Home.js:64]: nit: we disable this rule plenty of times throughout the codebase, can we disable it globally?

**Review by jdnmr — APPROVED** [review]
LG!

File: src/screens/Home/Home.js (hunk 2 above)
Comment (mdjastrzebski) [reply src/screens/Home/Home.js:64]: I'm not a fan of this rule, however it seems to be added by intent: https://github.com/pillarwallet/pillarwallet/pull/2213

@IAmKio maybe we could disable this rule it globally, wdyt?

**Pull request merged**